utils/hct/hctgettaef.py

```diff
@@ -8,10 +8,16 @@
 taef_dir = os.path.join(src_dir, "external", "taef")
 
 if not os.path.isdir(taef_dir):
-  os.mkdir(taef_dir)
+  os.makedirs(taef_dir)
+
+try:
+  urllib.urlretrieve(url, zipfile_name)
+except:
+  print("Unable to read file with urllib, trying via powershell...")
+  from subprocess import check_call, check_output
```

Comment:
I was using it to toggle between check_call and print(check_output). But I guess there's no point in keeping this around really.